lib/pymedphys/_experimental/streamlit/apps/wlutz/main.py

```diff
@@ -216,18 +220,129 @@ def main():
         advanced_mode,
     )
 
+    st.write("---")
+
     _presentation_of_results(wlutz_directory_by_date, advanced_mode)
 
 
+def _user_selected_angles(name, default_selection, default_tolerance):
+    name = name.capitalize()
+    default_selection = ", ".join(np.array(default_selection).astype(str))
+
+    st.write(f"### {name} filtering")
+
+    angles = st.text_input(f"{name} angles", default_selection)
+    angles = np.array(angles.split(",")).astype(float).tolist()
+    st.write(f"`{angles}`")
+
+    tolerance = st.number_input(f"{name} angle tolerance", 0, None, default_tolerance)
+
+    return angles, tolerance
+
+
+def _angle_filtering(database_table):
+    gantry_column, collimator_column = st.beta_columns(2)
+
+    default_gantry_angles = [-180, -135, -90, -45, 0, 45, 90, 135, 180]
```

Comment:
Useful to identify what coordinate system is in use.
I take it this is some native Elekta coordinate system because DICOM is usually non-negative modulo 360, although a certain vendor introduced a change to DICOM to allow values below zero and above 360 for some reason.